build_projects/dotnet-host-build/build.ps1

```diff
@@ -17,7 +20,10 @@ if($Help)
     Write-Host ""
     Write-Host "Options:"
     Write-Host "  -Configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
-    Write-Host "  -Architecture  <ARCHITECTURE>      Build the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
+    Write-Host "  -Architecture  <ARCHITECTURE>      Build on the specified architecture (x64 or x86 (supported only on Windows), default: x64)"
```

Comment:
And if this is only used within core-setup, I would suggest renaming this to `HostArchitecture` to make it obvious what it represents.